### PR TITLE
Restructure WSI Query chapter 

### PR DESCRIPTION
**Pull request by krOoze**
- split in sections
- move orphaned paragraphs
- concentrate behavior descriptions to
vkGetPhysicalDeviceSurfaceFormatsKHR (non-2)
- remove two paragraphs restating the obvious
- remove old hack

### CODE REVIEW

**Comment by stonesthrow** [comment]
On agenda for review.

File: chapters/VK_KHR_surface/wsi.txt [hunk 1]
```diff
@@ -1043,40 +1068,12 @@ encoded value (as stored in the image in memory).
 For most uses, the sRGB OETF is equivalent.
 ====
 
-
-An implementation supporting this extension indicates support for these
-color spaces via slink:VkSurfaceFormatKHR structures returned from
-flink:vkGetPhysicalDeviceSurfaceFormatsKHR.
-
-Specifying the supported surface color space when calling
-flink:vkCreateSwapchainKHR will create a swapchain using that color space.
```
Comment (krOoze) [inline chapters/VK_KHR_surface/wsi.txt]: BTW: these are the two removed paragraphs. I feel they do not add any useful information.

File: chapters/VK_KHR_surface/wsi.txt [hunk 2]
```diff
@@ -1010,18 +1043,10 @@ the following color spaces:
...
-The transfer functions are described in the "Transfer Functions" chapter of
-the <<data-format,Khronos Data Format Specification>>.
-
-// @@@ The ref page is ended earlier than in the original markup due to
-// asciidoctor issues described in internal MR 2201.
-endif::VK_EXT_swapchain_colorspace[]
-
---
-
-ifdef::VK_EXT_swapchain_colorspace[]
```
Comment (stonesthrow) [inline chapters/VK_KHR_surface/wsi.txt]: Does this ifdef need to stay to match the endif below?

Comment (krOoze) [reply chapters/VK_KHR_surface/wsi.txt]: No, it is merged with previous ifdef for which I removed endif at L1018

**Comment by oddhack** [comment]
@stonesthrow @tobski I'm going to go ahead and merge based on your reviews, but it will be helpful to add the "Approved To Merge" milestone in the future when the SI TSG signs off on a PR, as an unambigious signal.

**Comment by stonesthrow** [comment]
@oddhack I was looking for one more +1, and was going to mark this "Approved to merge" this morning. Sorry that is last minute agenda item.

**Comment by oddhack** [comment]
@stonesthrow OK, sorry for jumping the gun a bit. Fortunately worked out OK this time.

**Comment by cubanismo** [comment]
Ah, my page refreshed, and I see it was merged already.  Ignore me then :-)

**Pull request merged**